### DISCUSSION TITLE
关于我：一些经历

### DISCUSSION THREAD

**Discussion by ocoke** [ocoke/Biscus#7] [Archived]
我将我以前所有的博客文章存档，并在这里开始一个新的故事。
`[desc_end]`
## 编程

我是 Jun，此博客的作者。我三年前开始写博客。当时，我因为疫情不得不待在家里。我很无聊，想尝试一些新东西。在这之前，我学过 Python，因为尝试使用爬虫的机会，我无意中接触到了 Web 开发，并了解了一些 HTML 的基本信息。

然后，我创建了一些小项目，比如浏览器的起始页。其中一些现在甚至可以访问（但我不想访问）。我开始学习 JavaScript 和 CSS，因为我发现仅用 HTML 来构建现代网站是不够的。我从开源项目和其他文章中复制了大量代码到我的项目中。它工作了，但工作的不咋样。😂（比如我把所有的 `<script>` 标记都放在 `<head>` 标签里）。

之后，我想让其他人也能轻松访问我的网站，于是就买了一些域名。虽然当时的域名毫无意义（现在都过期了），但我感觉非常酷。因为买了这些域名，我开始在网站上使用一些现代框架：Hexo。Hexo 是一个静态网站生成器，有很多主题和插件。我更换了很多主题，试图为我的博客添加所有插件，使其变得 "强大"，结果速度变得非常慢。后来，为了让我的博客独一无二，我开始编写自己的主题。

我把所有 "美丽 "的元素放在一起，发现它们的组合是如此丑，页面变得很脏。我试着让它们统一起来，确保页面和谐。我不知道自己是如何提高审美的，也许是在浏览了很多不同的博客之后吧。

有人说，博客的性能非常重要，因为没有人愿意等 10 秒钟才能打开你的博客。好吧，我开始让我的博客变得 "精简 "和 "流畅"。这也花了我很多时间。我在这过程中探索了 Service Worker 和其他工具。我又在博客上加入上他们。同时，我也关心我的博客在中国大陆的可访问性（我没有 ICP 许可证🤣）。

几个月前（2023 年 3 月），我创建了你们现在访问的页面。大约两周前，我开始把我的新文章放到 GitHub Discussions 上，所以现在我不需要担心文章的存储问题了。

## 生活

在这些年里，我成为了一名高中生，我的思想发生了些许变化，我去了世界上不同的地方，看到了不同的地方和不同的人。

我喜欢摄影，每到一个新地方，我都会拍下很多照片。我打算把它们放到社交媒体上，也许以后会有机会。

顺便说一下，我的 16 Personalities 测试结果是 `INFP-T`。

这就是我的故事，现在我又开始了一个新博客，很高兴再次见到你。

## 未来

我想我会将我的所有文章存档在我的旧博客中，他们将能够访问，但不会出现在博客的信息流中。 博客还在开发中，我用自己的爱好来写这些文章，带着热爱。 我知道一个人不能用一些标签来描述，但这些是我的一些标签：

`#Developer`, `#Photographer`, `#Designer`, `#Coding`, `#Designing`, `#Chinese`, `#INFP`, `#Student`.

**Comment by stellaSalad** [comment]
INFP握爪，我也在搭建hexo，但是插件越多，bug越多，每天都在debug了

**Reply by ocoke** [reply]
握爪！
